assets/model_monitoring/components/src/shared_utilities/store_url.py

```diff
@@ -77,39 +76,58 @@ def _get_url(self, scheme=None, store_type=None, relative_path=None) -> str:
         return url
 
     def get_credential(self) -> Union[
-            str, ClientSecretCredential, AzureSasCredential, AzureMLOnBehalfOfCredential, None]:
+            str, ClientSecretCredential, AzureSasCredential, None]:
         """Get credential for this store url."""
         if not self._datastore:
+            from azure.ai.ml.identity import AzureMLOnBehalfOfCredential
+            print("Using AML OBO credential from StoreUrl.get_credential().")
```

Comment:
minor: line83, 94, 106 are same, can we distinguish them for better debugging purpose?

Reply:
Yes that makes sense. Added.